src/peer.c

```diff
@@ -24,24 +24,11 @@
  * SOFTWARE.
  */
 
-#include <arpa/inet.h>
-
-#include <stdarg.h>
-#include <stdint.h>
-#include <stdio.h>
-#include <unistd.h>
-
-#include "alloc.h"
 #include "compiler.h"
-#include "element.h"
-#include "fetch.h"
-#include "jet_string.h"
-#include "list.h"
-#include "log.h"
+#include INET
```

Comment:
What are is really required from <arpa/inet.h>? I think it just not necessary to include it.

So maybe the whole stuff around INET is not required.